dmscripts/generate_database_data.py

```diff
@@ -0,0 +1,48 @@
+import random
+import string
+import os
+from dmapiclient import DataAPIClient
+
+from typing import Optional
+
+USER_ROLES = [
+    'admin',
+    'admin-ccs-category',
+    'admin-ccs-sourcing',
+    'admin-manager',
+    'admin-framework-manager',
+    'admin-ccs-data-controller',
+    'buyer',
+    'supplier',
+]
```

Comment:
Nit: would an [enum](https://docs.python.org/3/library/enum.html) help here?

Reply:
I think there would be some benefits but, unless I'm misunderstanding, enum can't include dashes so I'd need to have something else as well to convert the enum value into the valid roles. I think that's adding too much complexity for what we gain

Reply:
Am I right the Enum would look like this?
```
class UserRoles(Enum):
    ADMIN = 'admin',
    ADMIN_CSS_SOURCING = 'admin-ccs-sourcing',
   .....
```